docs/source/intro/index.rst

```diff
@@ -0,0 +1,21 @@
+===============
+Getting started
+===============
+
+New to Django or MongoDB? Well, you came to the right place: read this material
+to quickly get up and running.
+
+.. toctree::
+   :maxdepth: 1
+
+   install
+   configure
+
+.. seealso::
+
+    If you're new to Django_, you might want to start by getting an idea of
+    what it's like. The :doc:`official Django tutorial
+    <django:intro/tutorial01>` is a great place to start. Once you understand
```

Comment:
We should also point to our Getting Started on the MongoDB docs as well. The Django Tutorial site won't make it clear how they should run the app using the library.